go-controller/pkg/cni/cniserver.go

```diff
@@ -212,12 +212,15 @@ func (s *Server) handleCNIRequest(r *http.Request) ([]byte, error) {
 
 func (s *Server) handleCNIMetrics(w http.ResponseWriter, r *http.Request) {
 	var cm CNIRequestMetrics
-
+	metrics.MetricCNIRequest.Inc()
 	b, _ := ioutil.ReadAll(r.Body)
 	if err := json.Unmarshal(b, &cm); err != nil {
```

Comment:
@martinkennelly qq on this codepath: do you intent to `.MetricCNIRequest.Inc()` even if the unmarshal below fails? Assuming so, I wonder if there is a case for bumping `metrics.MetricCNIRequestError` in cases when the unmarsling failed as well. Just a thought...
Otherwise, LGTM!